### ISSUE TITLE
Joomla 3.x and 4.x ACL - author access level 

### ISSUE THREAD

**Issue by sergeytolkachyov** [joomla/joomla-cms#35687]
### Steps to reproduce the issue
Both in J3 and j4 we have a preinstalled user groups and access permissions for this groups. On of them is Author. 
Author can do:
- site login
- create (any items)
**- edit own (items)**
Check this in Global configuration - Permissions.

Create a user with Author access level and try to create an article (com_content) via frontend.

### Expected result
We expect that Author can create it's own item, save and close it and go to drink a cup of coffee. Then after a while he can return and resume it's job. He goes to category, select his unfinished article and continue to edit and to save it.

### Actual result
Author goes to category where he created an article and sees nothing. He can't see it's own unpublished article.
If we add to Author ability to Edit (Global configuration - Permissions) or Edit sate - then Author can see unpublished articles. But also he can change article state and publish it without verification by Editor. Or he can edit all articles, not only own.

### System information (as much as possible)
Checked it on Joomla 3.10.2 and 4.0.3

**Comment by alikon** [comment]
shouldn't  this task can be done with workflow in J4 ? 
https://magazine.joomla.org/all-issues/march-2021/joomla-4-the-new-publishing-workflow-feature

**Comment by brianteeman** [comment]
@alikon workflow is optional - even if it is possible with workflows which I doubt

**Comment by sergeytolkachyov** [comment]
I tested a workflow on j4and find, that author can't see his own articles. I try to repeat this on j3 and found it too.

**Comment by brianteeman** [comment]
as I expected

**Comment by alikon** [comment]
there are some extensions?

**Comment by brianteeman** [comment]
No. It cant be done because as reported the core acl doesnt allow it. Try it and see.

**Comment by jeckodevelopment** [comment]
Would it need an additional "setting" in which the author can edit own articles only when they are still unpublished?

**Comment by brianteeman** [comment]
afaict yes that is the only way

**Comment by peteruoi** [comment]
I don't know the code but just from the words i believe it could be just a simple bug.
edit own permission should be the same with edit with an AND clause that only things created by me could be edited.
Can someone point me to the correct file that these permissions are defined? i 'm not a programmer but i understand algorithms.
Of course i am not sure about what i assume so plz be patient if it is something extremely difficult to fix.

**Comment by brianteeman** [comment]
This is just one of many issues which are basically the same thing and can only be resolved by additional ACL for edit.state.own

**Comment by sergeytolkachyov** [comment]
This issue has not been resolved. Is there any way to solve this problem now, before fixing it?
